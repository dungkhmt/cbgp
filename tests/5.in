20 20
20 30
12 5
10 0
7 2
17 10
8 5
15 11
15 0
17 9
19 7
16 5
8 2
7 18
4 9
7 1
9 14
18 0
13 0
6 2
7 3
14 17
18 1
11 10
12 4
5 2
2 0
13 4
14 11
12 13
5 13
18 2
1
0 0 21 1 20 3 30 0 14 6 16 4 21 4 27 0 18 5 13 7 9 7 11 9 15 5 12 4 12 8 10 10 21 9 11 7 20 0 23 4 
