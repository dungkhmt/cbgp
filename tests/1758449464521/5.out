(12, 5), (10, 0), (7, 2), (17, 10), (8, 5), (15, 11), (15, 0), (17, 9), (19, 7), (16, 5), (8, 2), (7, 18), (4, 9), (7, 1), (9, 14), (18, 0), (13, 0), (6, 2), (7, 3), (14, 17), (18, 1), (11, 10), (12, 4), (5, 2), (2, 0), (13, 4), (14, 11), (12, 13), (5, 13), (18, 2), 
numberIntersection=0 
minAngle=0.5112307122777899 
sumAngle=18.728798033797094 
minEdgeLength=2.0 
sumEdgeLength=520.7495701601576 
minDistance=0.018864566947613626 
sumDistance=13127.927743412698 

0.0000000000 0.5112307123 18.7287980338 2.0000000000 520.7495701602 0.0188645669 13127.9277434127
0: (14, 12),
1: (40, 5),
2: (38, 12),
3: (80, 80),
4: (26, 23),
5: (31, 16),
6: (37, 65),
7: (46, 3),
8: (35, 17),
9: (28, 25),
10: (21, 26),
11: (11, 39),
12: (28, 20),
13: (22, 17),
14: (24, 34),
15: (12, 30),
16: (37, 64),
17: (23, 26),
18: (35, 0),
19: (35, 80),
14 12 40 5 38 12 80 80 26 23 31 16 37 65 46 3 35 17 28 25 21 26 11 39 28 20 22 17 24 34 12 30 37 64 23 26 35 0 35 80 
0: (14, 12), 1: (40, 5), 2: (38, 12), 3: (80, 80), 4: (26, 23), 5: (31, 16), 6: (37, 65), 7: (46, 3), 8: (35, 17), 9: (28, 25), 10: (21, 26), 11: (11, 39), 12: (28, 20), 13: (22, 17), 14: (24, 34), 15: (12, 30), 16: (37, 64), 17: (23, 26), 18: (35, 0), 19: (35, 80), 
numberIntersection=0 minAngle=0.5112307122777899 sumAngle=18.72879803377954 minEdgeLength=2.0 sumEdgeLength=520.7495701601483 minDistance=0.018864566947613626 sumDistance=13127.92774341276 
Time taken: 13.117 s
